opm/input/eclipse/EclipseState/Grid/FieldProps.hpp

```diff
@@ -704,6 +704,9 @@ class FieldProps {
     void deleteMINPVV();
 
 private:
+    template<typename T>
+    static void update_global_from_local(Fieldprops::FieldData<T>& data,
+                                         const std::vector<Box::cell_index>& index_list);
```

Comment:
Why does this need to be a static member of the class?  It doesn't access any of the members of the class and could just as well be implemented as a regular function template in `FieldProps.cpp`.